apimonitor/tests/test_views.py

```
@@ -0,0 +1,261 @@
+        )
+        new_digest = baker.make(
+            AIDigest,
+            service=None,
+            created_at=timezone.now()
+        )
+        
+        response = self.client.get('/api/monitor/digests/latest/')
+        
+        assert response.status_code == status.HTTP_200_OK
+        assert response.data['id'] == new_digest.id
+        assert response.data['id'] != old_digest.id
```

Ruff (0.12.2)

Undefined name `old_digest`

(F821)